src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -38,9 +38,8 @@
 public final class CreateVariantIngestFiles extends VariantWalker {
     static final Logger logger = LogManager.getLogger(CreateVariantIngestFiles.class);
 
-    private PetTsvCreator petTsvCreator;
-    private VetTsvCreator vetTsvCreator;
-    private SampleInfoTsvCreator sampleInfoTsvCreator;
+    private PetCreator petCreator;
```

Comment:
Love the refactoring!  Maybe this could be "RefCreator" since it could be either Pet or ReferenceRanges ?